src/cf-core/src/cf-vars.less

```diff
@@ -12,62 +12,62 @@
 // https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
 
 // body
-@text:                          #212121; // $color-base
+@text:                   #212121; // $color-base
 
 // a
-@link-text:                     #0071bc; // $color-primary
-@link-underline:                #0071bc; // $color-primary
-@link-text-visited:             #4c2c92; // $color-visited
-@link-underline-visited:        #4c2c92; // $color-visited
-@link-text-hover:               #205493; // $color-primary-darker
-@link-underline-hover:          #205493; // $color-primary-darker
-@link-text-active:              #046b99; // $color-primary-darkest
-@link-underline-active:         #046b99; // $color-primary-darkest
+@link-text:              #0071bc; // $color-primary
+@link-underline:         #0071bc; // $color-primary
+@link-text-visited:      #4c2c92; // $color-visited
+@link-underline-visited: #4c2c92; // $color-visited
+@link-text-hover:        #205493; // $color-primary-darker
+@link-underline-hover:   #205493; // $color-primary-darker
+@link-text-active:       #046b99; // $color-primary-darkest
+@link-underline-active:  #046b99; // $color-primary-darkest
 
 // table
-@table-border:                  #5b616b; // $color-gray
+@table-border:           #5b616b; // $color-gray
 
 // thead
-@thead-text:                    @text;
-@thead-bg:                      #f1f1f1; // $color-gray-lightest
+@thead-text:             @text;
+@thead-bg:               #f1f1f1; // $color-gray-lightest
 
 // code
-@code-text:                     @text;
-@code-bg:                       #f1f1f1;
+@code-text:              @text;
+@code-bg:                #f1f1f1;
 
 
 // Sizing variables
 
-@base-font-size-px:             16px;
-@base-line-height-px:           22px;
-@base-line-height:              unit(@base-line-height-px / @base-font-size-px);
+@base-font-size-px:      16px;
+@base-line-height-px:    22px;
+@base-line-height:       unit(@base-line-height-px / @base-font-size-px);
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/499/commits/61606fba9e694fbe43965695937f89318c0f7e16